### ISSUE TITLE
Issue when compiling IteRace

### ISSUE THREAD

**Issue by rvalde28** [cos/IteRace#8]
I am having trouble in trying to get the last step to go. It returns the error: 
[[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: University of Illinois#util_2.10;0.1: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn] 	Note: Unresolved dependencies path:
[warn] 		University of Illinois:util_2.10:0.1 (/home/ruben/Downloads/IteRace/build.sbt#L20-29)
[warn] 		  +- University of Illinois:iterace_2.10:0.5
[info] Wrote /home/ruben/Downloads/IteRace/target/scala-2.10/iterace_2.10-0.5.pom
[error] sbt.librarymanagement.ResolveException: unresolved dependency: University of Illinois#util_2.10;0.1: not found
 I was wondering if you could help me on this issue so that maybe I can get this up and running. I would like to see it in action.

Thank you

**Comment by cos** [comment]
hi @rvalde28, did you follow the instructions here? 
https://github.com/cos/IteRace/blob/master/README.md

**Comment by rvalde28** [comment]
Yes, I followed them step by step. For some of the programs, I had to change the values for addSbtPlugin for sbt eclipse from 2.4.0 to 5.3.4 since 2.4.0 is no longer supported. I was able to get WALA and WALAFacade installed and all of the other engines that it depends on. I get as far as compiling the Util folder on your repo, but compiling IteRace returns an error.
